WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -55,7 +56,10 @@ public void onCreate(Bundle savedInstanceState) {
         }
 
         if (savedInstanceState == null) {
-            PeopleTable.deletePeopleForLocalBlogIdExceptForFirstPage(blog.getLocalTableBlogId());
+            // only delete cached people if there is a connection
+            if (NetworkUtils.isNetworkAvailable(this)) {
```

Comment:
I think it would make sense to use `NetworkUtils.checkConnection()` here instead, since that shows a toast letting the user know there's no connection. Without this, the user may be stuck looking at an empty list with no indication of why it's empty (and the message is useful even if cached users are shown).